packages/admin/admin-stories/src/admin/alerts/Alerts.tsx

```diff
@@ -0,0 +1,21 @@
+import { AdminComponentAlerts } from "@comet/admin";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+function Story() {
+    return (
+        <>
+            <AdminComponentAlerts severity="info" title="Title" text="Notification Text" />
+            <br />
+            <AdminComponentAlerts severity="warning" title="Title" text="Notification Text" />
+            <br />
+            <AdminComponentAlerts severity="error" title="Title" text="Notification Text" />
+            <br />
+            <AdminComponentAlerts severity="success" title="Title" text="Notification Text" />
+            <br />
+            <AdminComponentAlerts severity="success" text="Notification Text" />
+        </>
```

Comment:
You could do this with a `Stack` so you don't need all the `br`s 🙂 

```suggestion
        <Stack spacing={4}>
            <AdminComponentAlerts severity="info" title="Title" text="Notification Text" />
            <AdminComponentAlerts severity="warning" title="Title" text="Notification Text" />
            <AdminComponentAlerts severity="error" title="Title" text="Notification Text" />
            <AdminComponentAlerts severity="success" title="Title" text="Notification Text" />
            <AdminComponentAlerts severity="success" text="Notification Text" />
        </Stack>
```